src/substrate-lib/utils.js

```diff
@@ -0,0 +1,19 @@
+const Big = require('toformat')(require('big.js'));
```

Comment:
We've used `import` rather than `require` in other places. Is that possible here? I hope that's not a dumb question. I've never seen two requires in a single expression like this.

Reply:
Let me work on it to convert them to import.

But there isn't much magic here. This `require('toformat')` is returning an obj, and then `require('big.js')` is returning another object. So eventually, it is just like `Obj1(Obj2)`. `Obj1` is a decorator to `Obj2` giving extra formatting functionality. `require` and `import` are quite equivalent in functionality, with `require` more commonly seen in node (server-side js dev). [A lot of discussion on commonJS require vs es6 import](https://stackoverflow.com/questions/31354559/using-node-js-require-vs-es6-import-export). But one big difference is `require` allow dynamic string loading while es6 import doesn't. So `require` is needed in loading the env-specific json file.

[Original syntax by the lib author seen here](https://github.com/MikeMcl/toFormat).

Reply:
Updated with `import`